src/containers/About/About.js

```diff
@@ -0,0 +1,182 @@
+    }
+
+    if (this.state.isNotFinished) {
+      this.setState({
+        error: errorMessage,
+        isNotFinished: false
+      });
+    }
+  }
+
+  async fetchDashboardInfo() {
+    const dash = await getInstallProperties();
+    this.setState({
+      dashboardInfo: dash,
+      isLoaded: true
+    });
+  }
+
+  render() {
+    const { intl, loading } = this.props;
+    if (this.state.dashboardInfo !== null && this.state.isLoaded === true) {
+      const initialHeaders = [
+        {
+          key: 'property',
+          header: intl.formatMessage({
+            id: 'dashboard.tableHeader.Property',
+            defaultMessage: 'Property'
+          })
+        },
+        {
+          key: 'value',
+          header: intl.formatMessage({
+            id: 'dashboard.tableHeader.Value',
+            defaultMessage: 'Value'
+          })
+        }
+      ];
+
+      const initialRows = [];
+      let data = '';
+      const errorsToDisplay = [];
+
+      if (
+        this.state.dashboardInfo.InstallNamespace !== '' &&
+        this.state.dashboardInfo.InstallNamespace !== 'undefined'
+      ) {
+        data = {
+          id: `InstallNamespace`,
+          property: 'Install Namespace',
+          value: this.state.dashboardInfo.InstallNamespace
+        };
+        initialRows.push(data);
+      } else {
+        errorsToDisplay.push('Install Namespace');
+      }
+
+      if (
+        this.state.dashboardInfo.DashboardVersion !== '' &&
+        this.state.dashboardInfo.DashboardVersion !== 'undefined'
+      ) {
+        data = {
+          id: `DashboardVersion`,
+          property: 'Dashboard Version',
+          value: this.state.dashboardInfo.DashboardVersion
+        };
+        initialRows.push(data);
+      } else {
+        errorsToDisplay.push('Dashboard');
+      }
+
+      if (
+        this.state.dashboardInfo.PipelineVersion !== '' &&
+        this.state.dashboardInfo.PipelineVersion !== 'undefined'
+      ) {
+        data = {
+          id: `PipelineVersion`,
+          property: 'Pipeline Version',
+          value: this.state.dashboardInfo.PipelineVersion
+        };
+        initialRows.push(data);
+      } else {
+        errorsToDisplay.push('Pipeline');
+      }
+
+      if (errorsToDisplay.length !== 0) {
+        this.setErrorState(errorsToDisplay);
+      }
+
+      return (
+        <>
+          {this.state.error !== '' && (
+            <InlineNotification
+              kind="error"
+              title={intl.formatMessage({
+                id: 'dashboard.displayVersion.error',
+                defaultMessage: 'Error getting version'
+              })}
+              subtitle={this.state.error}
+              lowContrast
+            />
+          )}
+          <h1>About</h1>
+          <Table
+            headers={initialHeaders}
+            rows={initialRows}
+            loading={loading}
+          />
+        </>
+      );
+    }
+
+    return (
+      <>
+        <h1>About</h1>
+      </>
```

Comment:
we should have a loading state here instead of a blank page with just the title

can we show the table with inline loading indicators for the values? or even just a blank table loading state as we have on other pages

Reply:
I have changed it to displaying the table headers
Once it gets the info it does update the table and display it